pkgs/unified_analytics/lib/src/constants.dart

```diff
@@ -82,13 +82,13 @@ const int kLogFileLength = 2500;
 const String kLogFileName = 'dart-flutter-telemetry.log';
 
 /// The current version of the package, should be in line with pubspec version.
-const String kPackageVersion = '5.8.5';
+const String kPackageVersion = '5.8.6';
 
 /// The minimum length for a session.
 const int kSessionDurationMinutes = 30;
 
 /// Name for the json file where the session details will be stored.
-const String kSessionFileName = 'dart-flutter-telemetry-session.json';
+const String kSessionFileName = 'dart-flutter-telemetry.session';
```

Comment:
This rename for the file will mean that the old one will remain untouched... should we check for the existence of this file and remove it?

Reply:
Why do we need to rename the file?

Reply:
This was assuming that I changed the format of the data, but if we kept the file in json format and just removed the `last_ping` key, we can keep the same name